project-resources/community-health-files-guide.md

```diff
@@ -0,0 +1,61 @@
+# Guide to Community Health Files for OpenJS Foundation Projects
+
+Community health files are documents related to project collaboration. Most commonly, these include:
+
+- `CODE_OF_CONDUCT.md`
+- `CONTRIBUTING.md`
+- `GOVERNANCE.md`
+- `SECURITY.md`
+
+> [!NOTE]
+> See the [full list](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types) of supported default community health files
+
+## Default Community Health Files
+
+Instead of adding these files to every repository within an organization, we can add them only once to a special organization repository which must be named `.github`. GitHub will automatically use the community health files specified at this location by default for all repositories within the organization.
+
+The filenames must match [what is expected by GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file#supported-file-types) and must be in one of these locations:
+
+- The root of the `.github` repository
```

Comment:
Let's recommend the best option here. The idea of a best practice guide isn't to inform about all possible options, but to make it easy for folks to do the right thing.

Reply:
I think this should read "in the root of the GitHub repository". I'm not sure what a `.github` repository is.

Reply:
Sorry, nevermind, that's a special repository. Is that a necessity? For instance, while jQuery may have lots of repositories, the core repo is obviously the main one and I'd only want to put these files there.

Reply:
it's a github repository named `.github`.

Reply:
I think you missed my next comment. I wonder if the wording could change here to account for those that don't need or want a `.github` repository.

Reply:
It's certainly a necessity for things like a code of conduct to be in every repo - without a `.github` repo, you'd have to repeat them in every repo.

Reply:
1. That's fine for those with a low number of active repos
2. A code of conduct is required for the organization. The absence of the file in one of the repos does not mean the organization doesn't prescribe to a code of conduct, especially if the main repo has it.

Reply:
I suppose that's technically true, but participants in one of the other repos won't be able to become aware of the CoC if it's not in one of the github-prescribed locations for it. iow, there's no "main" repo in a generic sense in a github org, so a user wouldn't necessarily have a way of knowing which one it is.

Reply:
either way, these are guidelines, not mandates, so it's fine if jquery doesn't want to follow them - in general, orgs really should have a .github repo.

Reply:
I still see some edge cases where this reads weird, but I'll set this up for jQuery

Reply:
Regarding the original ask, I think the best option is top-level.

Reply:
> those that don't need or want a .github repository.

a major goal of this guide is that all orgs should have/use a `.github` repo

Reply:
> a major goal of this guide is that all orgs should have/use a .github repo

I agree that this is the easiest way to get at the outcome we care about, which is that every project has all of the right files in the right places.

I think framing it that way helps understand the benefits of the `.github` repo (which I don't think are clear to everyone): it acts kind of like a dynamic repo template system (or like properties of the prototype of an JS object, to use an analogy that should speak more clearly to folks in this crowd).